packages/utils/src/tests/utils.test.jsx

```diff
@@ -0,0 +1,20 @@
+import { hasFeatureFlagEnabled } from '../utils';
+
+describe('hasFeatureFlagEnabled', () => {
+  const { location } = global;
+
+  beforeAll(() => {
+    delete global.location;
+    global.location = { search: '?feature=test1&feature=test2' };
+  });
+
+  afterAll(() => {
+    global.location = location;
+  });
+
+  test('properly determines feature flags from query params', () => {
```

Comment:
superduperubernit: does using it(' make more sense here?

Reply:
Follow-up from Slack. `test` and `it` are aliases in Jest. The only difference between them is which one makes the most readable test name as a sentence. Opting to leave as is.